internal/Proyecto.go

```diff
@@ -0,0 +1,16 @@
+package internal
+
+type Tarea uint8
+
+const (
+	Facil   Tarea = 0
```

Comment:
Esto continúa aquí, a pesar de que hablamos ayer de que no había nada en la historia de usuario que apoyara su existencia.

Reply:
En la HU pone que los encargados tienen tareas más importantes que hacer, lo que los empleados también tienen que hacer tareas relacionadas con el proyecto. Con esto se ve si una tarea es importante o no, para poder asignar más o menos días de vacaciones.

Reply:
No, no se ve ni eso es relevante a la lógica de negocio. ¿De qué forma influye que el encargado tenga una tarea de dificultad "muy difícil" para que yo pida 3 días de vacaciones en abril? Os rogaría que para optimizar el tiempo de todo el mundo implicado, tratárais de entender un poco mejor tanto el problema que tratáis como las objeciones que hacemos a vuestro trabajo relacionado con el mismo.

Reply:
En el momento que lo pensé tiene bastante sentido ya que se le asigna un número, en el que va a tener una serie de tareas que van a poder multiplicar de manera mayor o menor el hecho de que las tareas sean más o menos complicadas ya que eso requerirá más o menos tiempo.